diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5b53f31..a5d3986 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -108,9 +108,15 @@ else()
     )
 endif()
 
-if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR AND NOT BOOST_URL_IN_BOOST_TREE)
+if(NOT BOOST_URL_IN_BOOST_TREE)
     set_target_properties(boost_url PROPERTIES EXPORT_NAME url)
-    install(TARGETS boost_url EXPORT boost_url_targets)
+    install(
+	    TARGETS boost_url
+	    EXPORT boost_url_targets
+	    ARCHIVE DESTINATION lib
+	    LIBRARY DESTINATION lib
+	    RUNTIME DESTINATION bin
+    )
 
     install(EXPORT boost_url_targets
         FILE boost_url-targets.cmake
